### PR TITLE
chore: fix minor compile warnings

### PR DESCRIPTION
**Pull request by XuNeo**
A second try to fix all warnings...

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by kisvegabor — DISMISSED** [review]

File: Kconfig [hunk 1]
```diff
@@ -1551,7 +1551,7 @@ menu "LVGL configuration"
 			default 9 # LVGL_VERSION_MAJOR
 		config LVGL_VERSION_MINOR
 			int
-			default 2 # LVGL_VERSION_MINOR
+			default 3 # LVGL_VERSION_MINOR
```
Comment (kissa96) [inline Kconfig:1573]: I am not sure about minor rev being 3. @kisvegabor can you check this?

Comment (kisvegabor) [reply Kconfig:1573]: As we are developing v9.3 in master it should be correct.

**Review by FASTSHIFT — DISMISSED** [review]

**Comment by XuNeo** [comment]
Need to fix some newly introduced warnings from nuttx.

```
CC:  semaphore/sem_tickwait.c /home/neo/projects/nuttx/apps/graphics/lvgl/lvgl/demos/ebike/translations/lv_i18n.c:10: warning: "UNUSED" redefined
   10 | #define UNUSED(x) (void)(x)

```

**Review by kisvegabor — APPROVED** [review]
lv_i18n.c is auto-generated, but we should update its format for sure.

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**